projects/wallet-template/STEP-BY-STEP-GUIDE.md

```diff
@@ -0,0 +1,134 @@
+# Step by Step Guide
+
+To add light client functionality to your dapp, follow this step by step guide.
+
+## Requirements
+
+- Manifest V3
```

Comment:
also, a link to relevant docs would be nice.

Reply:
Assuming chrome & firefox are the only relevant browser extensions then yes. this is because we need background services workers which are only avaliable in manifest v3. i will add it to the docs.

https://developer.chrome.com/docs/extensions/develop/migrate/to-service-workers